--- a/configure.ac
+++ b/configure.ac
@@ -1917,7 +1917,6 @@
   lib/Makefile
   plugins/Makefile
   lib/tests/Makefile
-  plugins-root/Makefile
   plugins-scripts/Makefile
   plugins-scripts/utils.pm
   plugins-scripts/utils.sh

--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = gl tap lib plugins plugins-scripts plugins-root po @PERLMODS_DIR@
+SUBDIRS = gl tap lib plugins plugins-scripts po @PERLMODS_DIR@

 EXTRA_DIST = config.rpath \
 	ABOUT-NLS ACKNOWLEDGEMENTS AUTHORS CODING ChangeLog FAQ NEWS \
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -27,6 +27,6 @@
 #AM_CFLAGS = -Wall
 
-libexec_PROGRAMS = check_apt check_cluster check_disk check_dummy check_http check_load \
+libexec_PROGRAMS = check_apt check_cluster check_disk check_dummy check_http \
 	check_mrtg check_mrtgtraf check_ntp check_ntp_peer check_nwstat check_overcr check_ping \
 	check_real check_smtp check_ssh check_tcp check_time check_ntp_time \
